Release notes/RN220216 copy.md

```diff
@@ -0,0 +1,27 @@
+# 220216 (AzSK.ADO v.1.16.0)
+
+## Feature Updates
+
+* ``ADO_AgentPool_DP_No_Secrets_In_Capabilities:`` Added automated fixed for User  defined capabilities.
```

Comment:
Automated bulk remediation extended for:
ADO_AgentPool_DP_No_Secrets_In_Capabilities
ADO_AgentPool_DP_Enable_Auto_Update

Following the format from previous release notes